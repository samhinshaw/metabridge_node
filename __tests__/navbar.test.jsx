import React from 'react';
import { shallow } from 'enzyme';
import NavBar from '../components/navbar';

const mockRouter = {
  pathname: '/'
};

test('Search renders correctly', () => {
  const component = shallow(<NavBar router={mockRouter} />);
  expect(component).toMatchSnapshot();
});

// test('Search should render correct amount of shows', () => {
//   const component = shallow(<UnwrappedSearch shows={preload.shows} searchTerm="" />);
//   expect(component.find(ShowCard).length).toEqual(preload.shows.length);
// });
