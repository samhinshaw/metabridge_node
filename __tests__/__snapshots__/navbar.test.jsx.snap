// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <withRoute(NavBar)
    router={
      Object {
        "pathname": "/",
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "router": Object {
        "pathname": "/",
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "router": Object {
          "pathname": "/",
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
